contracts/controller/StorageEnabled.sol

```diff
@@ -79,4 +79,18 @@ contract StorageEnabled {
     Storage(storageAddr).setUInt('Power', 'authorizedPower', _newAuthorizedPower);
   }
 
+
+  function downs(address _user) constant public returns (uint256 total, uint256 left, uint256 start) {
+    uint256 rawBytes = Storage(storageAddr).getBal('PowerDown', _user);
+    start = uint96(rawBytes >> (8));
+    left = uint96(rawBytes >> (8 + 96));
+    total = uint96(rawBytes >> (8 + 96 + 96));
+    return;
+  }
+
+  function _setDownRequest(address _holder, uint256 total, uint256 left, uint256 start) internal {
+    uint256 result = (start << 8) + (left << (96 + 8)) + (total << (96 + 96 + 8));
```

Comment:
since you are not using version byte, we can just drop it and make `start` to be 8 bytes long.

Reply:
I think there is a problem with this line. The number of bits are off.

we have to pack it like this `<total-12b><left-12b><start-8b>`, so the proper experssion will be some thing like this:
```
uint256 result = uint64(start) + (left << 64) + (total << (64 + 96));
```

`#downs` method should be changed accordingly.

@bkcninja please take a look as well

Reply:
Snippet to verify:

```
pragma solidity ^0.4.11;

contract Test {

    function pack(uint a, uint b, uint c) returns (uint256) {
        return uint64(a) + (b << 64) + (c << (96 + 64));
    }
    
    function unpack(uint r) returns (uint start, uint left, uint total) {
        start = uint64(r);
        left = uint96(r >> (64));
        total = uint96(r >> (96 + 64));
        return;
    }
    
    function test() {
        uint total = 20000000000002;
        uint left = 10000000000001;
        uint start = now;

        uint packed = pack(total, left, start);
        var (total1, left1, start1) = unpack(packed);
        assert(total == total1);
        assert(left == left1);
        assert(start == start1);
        
    }

}
```